chapters/background.tex

```diff
@@ -1,11 +1,12 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-TensorFlow is a open source Machine Learning library developed by Google.
+The renaissance of machine learning in the 2010s outclassed this approach because it lacks the ability to detect emotions/smiles.
```

Comment:
I would just skip this sentence as well (sorry, I know... hard work)

Reply:
This sentence is removed in the next commit